.github/workflows/add-greeting-to-issue.yml

```diff
@@ -24,6 +24,8 @@ jobs:
 
           Newcomers, we're excited to have you on board. Start by exploring our [Contributing](https://github.com/JabRef/jabref/blob/main/CONTRIBUTING.md) guidelines, and don't forget to check out our [workspace setup guidelines](https://devdocs.jabref.org/getting-into-the-code/guidelines-for-setting-up-a-local-workspace) to get started smoothly.
 
```

Comment:
we currently habe no formal code of conduct... not sure if we should invest time to create one...